### PR TITLE
KAN-5-input

### PR DESCRIPTION
**Pull request by stasfilippov**

### CODE REVIEW

File: src/components/ui/textField/textField.module.scss [hunk 1]
```diff
@@ -0,0 +1,108 @@
+@mixin reset {
```
Comment (doroshkent) [inline src/components/ui/textField/textField.module.scss:1]: what is it here for if not used?

File: src/components/ui/textField/textField.module.scss [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+  display: flex;
+  align-items: center;
+  top: 50%;
+  right: 12px;
+  transform: translateY(-50%);
+  border: none;
+  background: transparent;
+
+  svg {
+  }
```
Comment (doroshkent) [inline src/components/ui/textField/textField.module.scss:46]: empty rule

File: src/components/ui/textField/textField.stories.ts [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+  //       'h3',
+  //       'h4',
+  //       'link1',
+  //       'link2',
+  //       'overline',
+  //       'subtitle1',
+  //       'subtitle2',
+  //     ],
+  //   },
+  // },
```
Comment (doroshkent) [inline src/components/ui/textField/textField.stories.ts:24]: do we need that comment?

File: src/components/ui/textField/textField.tsx [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+    if (inputRef.current) {
+      inputRef.current.value = ''
+    }
+  }
+  const handleFocus = () => {
+    setIsActive(true)
+  }
+  const handleBlur = () => {
+    setIsActive(false)
+  }
```
Comment (doroshkent) [inline src/components/ui/textField/textField.tsx:74]: according to 5th sprint it is better to name callbacks in a similar way. so it should be either "onInputChange" "onClearInput" "onFocus" or "changeHandler" "focusHandler" "clearHandler" etc. more ideas here https://samurai.it-incubator.io/pc/video-content/watch/6647de0309be72a38156d089 at 50:30

**Review by doroshkent — APPROVED** [review]

**Review by VladZimin — APPROVED** [review]

**Pull request merged**